sly-hello-world-autoloads.el

```diff
@@ -1,10 +1,33 @@
+;;; sly-hello-world-autoloads.el --- automatically extracted autoloads
+;;
+;;; Code:
+
+(add-to-list 'load-path (directory-file-name
+                         (or (file-name-directory #$) (car load-path))))
+
 
-;;;### (autoloads nil "sly-hello-world" "sly-hello-world.el"
-;;;;;;  (21964 37122 0 0))
+;;;### (autoloads nil "sly-hello-world" "sly-hello-world.el" (0 0
+;;;;;;  0 0))
 ;;; Generated autoloads from sly-hello-world.el
 
 (add-to-list 'sly-contribs 'sly-hello-world 'append)
 
+(if (fboundp 'register-definition-prefixes) (register-definition-prefixes "sly-hello-world" '("sly-hello-world")))
+
 ;;;***
+
+;;;### (autoloads nil "sly-hello-world-tests" "sly-hello-world-tests.el"
+;;;;;;  (0 0 0 0))
+;;; Generated autoloads from sly-hello-world-tests.el
 
-(provide 'sly-hello-world-autoloads)
+(if (fboundp 'register-definition-prefixes) (register-definition-prefixes "sly-hello-world-tests" '("sly-hello-world-basic-test")))
+
+;;;***
+
+;; Local Variables:
+;; version-control: never
```

Comment:
version-control: never isn't the case here. IOW we're using the automatic generation facilities to make a versioned file. If it's not much trouble, remove it, otherwise it won't hurt much either.

Reply:
Mon Jul  1 13:34:15 BST 2019

Removed.